src/game.cpp

```diff
@@ -3562,6 +3562,21 @@ bool Game::playerSpeakTo(Player* player, SpeakClasses type, const std::string& r
 		type = TALKTYPE_PRIVATE_FROM;
 	}
 
+	if (!player->isAccessPlayer() && !player->hasFlag(PlayerFlag_IgnoreSendPrivateCheck)) {
```

Comment:
The check is probably redundant now. `isAccessPlayer` checks if the player has _any_ flag, then it checks if it has a specific flag - which makes the first operand true anyway.
```suggestion
	if (!player->hasFlag(PlayerFlag_IgnoreSendPrivateCheck)) {
```